### ISSUE TITLE
[FEATURE] [Scaneingang] Datum nur optional ergänzen 

### ISSUE THREAD

**Issue by stm9x9** [jlawyerorg/j-lawyer-org#2490]
Aktuell wird bei Scaneingängen dem Dateinnamen im Scanordner das aktuelle Datum vorangestellt. In meiner Kanzlei wird bereits beim Scannen ein sinnvoller Dateiname vergeben, einschließlich des Datums des Dokuments. Daher müssen wir hier für jedes Dokument manuell das vorne wieder löschen. Und es ist nicht nur die zusätzliche Arbeit, ich bekomme das Zittern in den Augen, wenn ich die richtige Stelle in den 20 Ziffern finden will :-)

![image](https://github.com/user-attachments/assets/75ba7d60-a109-4b21-b1f1-976edad06318)

Hilfreich wäre es, wenn man oben wie die anderen Einstellungen auch einstellen und vorgeben könnte, ob das Datum wie aktuell vorangestellt werden soll oder nicht.

**Comment by j-dimension** [comment]
Ist das nicht direkt durch eine Einstellung am Scanner / in der Scansoftware lösbar?

**Comment by stm9x9** [comment]
Nein, denn jl will ja das aktuelle Datum (bei Bearbeitung des Scaneingangs in jl) hinzufügen. Das kann ja der Scanner nicht verhindern. Wenn ich am 14.7. in jl den gescannten Bescheid vom 10.7. importiere, dann will ich ja, daß das Dokument 2024-07-10 Bescheid heißt (und so benenne ich es auch in der Scansoftware). Und nicht 2024-07-14_15-01-32_2024-07-10 Bescheid.

**Comment by j-dimension** [comment]
Ich wollte darauf hinaus, dem Scanner das Anbringen eines Datumspräfixes auszureden. Nur der j-lawyer würde dann einen solchen Präfix anbringen.

**Comment by stm9x9** [comment]
Das Präfix bringe ich ja selber im Scanvorgang an, während ich das Dokument passend zum Inhalt benenne. Wenn ich dies wegläße, dann machte jl das daraus: 2024-07-14_15-01-32_Bescheid. Aber der Bescheid ist ja nicht vom 14.7. Ehrlich gesagt, fehlt mir die Phantasie, welchen Mehrwert überhaupt der Zeitstempel des Import-Vorgangs in jl bringt. Aber weil ich ja niemandem einen use case wegnehmen will, dachte ich halt an ein optionales Anbringen.

**Comment by iradraconis** [comment]
Das Datum, welches im Scaneingang angebracht wird, ist bei uns auch Datum des Eingangs. Wenn Dateien im Ordner dann nach Namen sortiert werden, ergibt sich eine sinnvolle Reihenfolge im Sinne einer Chronologie der Akte. 

Wir benennen dann die Dateien individuell: 2024-07-12_Bussgeldbescheid_vom_5-7-2024 oder so.

**Comment by stm9x9** [comment]
und wie sortiert ihr dann nach der Chronologie der Bescheide/Schriftsätze? Chronologie der Akte bekomme ich ja über die Sortierung nach Erstellungsdatum? (BTW ich habe schon in der Papierakte einen Bescheid vom 5.7. unter dem 5.7. und nicht dem Datum des Eingangs in der Kanzlei (ob nun von Gegner oder vom Mandant) einsortiert. Eingang belegt der Eingangsstempel. Haben das echt andere in Papier anders gehandhabt? Was es nicht alles gibt :-)) Aber deshalb sag ich ja, optional, ich möchte ja niemandem was wegnehmen.

**Comment by iradraconis** [comment]
Es lässt sich in JLawyer auch ein Datum für ein Dokument setzen. Dann könnte nach Datum sortiert werden. Man könnte auch die gewünschten Schreiben könnte man auch in einen Unterordner legen zwecks Übersichtlichkeit (nur den Ordner einblenden) und Darstellung einer Chronologie.

**Comment by j-dimension** [comment]
Aber die Scansoftware weiß ja auch nicht, wann der Eingang war? Wird immer am Tag des Eingangs gescannt?

**Comment by steboe1** [comment]
Nein, das macht der MA manuell. Posteingang z.B. am Freitag mit Eingangsstempel durch mich, Ablage in Postfach des MA mit Importberechtigung, dieser erledigt e aber erst am nächsten Werktag z.B. am Montag

**Comment by j-dimension** [comment]
Wo ist dann der Unterschied / Produktivitätsgewinn zwischen "manuell in der Scansoftware" vs. "manuell im j-lawyer-Scaneingang"?

**Comment by steboe1** [comment]
weil in jl das Datum automatisch dazu kommt und wieder gelöscht werden muss

![image](https://github.com/user-attachments/assets/316dcc41-4263-4ad9-aa31-6d30c2ff374a)

**Comment by steboe1** [comment]
![image](https://github.com/user-attachments/assets/4129ef6a-ce32-4f4f-8561-8b7546a205ed)

**Comment by j-dimension** [comment]
@steboe1 ich verstehe was weg soll. Nur: wenn man beim Scannen das 2023-03-20 weglässt, entfällt dort der manuelle Aufwand, und man leistet ihn dann beim Verakten im j-lawyer. Ist das nicht ein Nullsummenspiel, mit geringfügiger Anpassung des Ablaufs?

**Comment by steboe1** [comment]
@j-dimension nein sehe ich nicht so. lj fügt ein Datum ein welches an dieser Stelle nicht sein darf (zumindest nach meiner Vorstellung :-) ) wenn Eingangstag nicht gleich Tag des Imports ist.  Es hilft mir doch nicht das Datum im Eingangsscan wegzulassen wenn jl mir das Datum des Tages des Imports in jl aufzwingen will. 
Vll bin ich da auch ein wenig stur, grundsätzlich finde ich die Importfunktion in jl sehr gut. 
Wenn der Aufwand unverhältnismäßig hoch ist verstehe ich das. 
Da ich nicht der Entwickler bin fehlt mir das Detailwissen.
An diesem Punkt würde ich das dann abbrechen um keine Endlosdiskussion loszubrechen.

**Comment by stm9x9** [comment]
Wenn ich @steboe1 richtig verstehe, dann will er - was ich als eine Option auch verstehe - das Datum des Scans am Dokument haben. Das leistet die Scansoftware automatisch. Das Datum vom Import aber ist das falsche für ihn.

Aber auch für mich macht einen Unterschied: Wenn ich scanne, dann scanne ich Dokument für Dokument. In dem Workflow liegt mir daher immer das gerade aktuell gescannte einmal vor, ich sehe es und kann es passend benennen. Wenn ich dann aber mehr als ein Dokument im Scaneingang liegen habe und per Massenimport in jl hole, dann müßte ich jedes Dokument erstmal öffnen, um überhaupt zu wissen, was ich da oben in dem Beispiel eingebe (um nicht alles doppelt zu machen, müßte ich ja auch den eigentlichen Namen erstmal eingeben und dafür wissen, was drin ist).

**Comment by j-dimension** [comment]
> Das leistet die Scansoftware automatisch.

@stm9x9 Die nimmt doch auch nur das Datum des Scanvorgangs (also das aktuelle Datum), oder?

**Comment by stm9x9** [comment]
dann habe ich steboe1 falsch verstanden, dann gilt nur mein zweiter Absatz :-)

**Comment by steboe1** [comment]
@stm9x9 Ich möchte eigentlich kein Datum beim Import vorgeschlagen bekommen weil das bereits meine Scanner-Software macht. Meine Post wird nicht zwingend tagesaktuell im Scaneingang von JL verarbeitet daher stört mich das ergänzen eines Datums eher.

**Comment by stm9x9** [comment]
wenn das in allen Massenimport-Dialogen umgesetzt würde, dann wäre auch das Importieren von Behördenakten wesentlich einfacher, weil das Datum ja die Zuordnung in der xml-Datei für den Justiz Viewer zerstört.
